cdap-ui/app/cdap/components/Tags/Tag/index.js

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright © 2017 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import React, {PropTypes, Component} from 'react';
+import SpotlightModal from 'components/SpotlightSearch/SpotlightModal';
+require('./Tag.scss');
+
+export default class Tag extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      searchModalOpen: false,
+    };
+
+    this.toggleSearchModal = this.toggleSearchModal.bind(this);
+  }
+
+  toggleSearchModal() {
+    this.setState({
+      searchModalOpen: !this.state.searchModalOpen
+    });
+  }
+
+  render() {
+    return (
+      <span>
+        {
+          this.props.scope === 'SYSTEM' ?
```

Comment:
might be able to simplify this tertiary operation by using `classnames` to conditionally add the user or system-tags class, then simply show or hide the close button